guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,1280 @@
+
+### InviteGuild
+
+Provides API support for the partial Guild data available from an `Invite`.
+
+### InviteStageInstance
+
+Provides API support for bots to inviting users to Stage Instances.
+
+### Message
+
+#### Message#awaitMessageComponent
+
+A shortcut method to create a promisified `InteractionCollector` which resolves to a single `MessageComponentInteraction`.
+
+#### Message#createMessageComponentCollector
+
+A shortcut method to create an `InteractionCollector` for components on a specific message.
+
+#### Message#crosspostable
+
+Checks permissions to see if a Message can be crossposted.
+
+#### Message#edit
+
+Editing and/or removing attachments when editing a Message is now supported.
+
+#### Message#fetchReference
+
+Provides support for fetching the Message referenced by `Message#reference`, if the client has access to do so.
+
+#### Message#react
+
+Now supports both `<:name:id>` and `<a:name:id>` as valid inputs.
+
+#### Message#stickers
+
+A Collection of Stickers which were in the message.
+
+### MessageActionRow
+
+A builder class which makes constructing action row type Message Components easy.
+
+### MessageAttachment
+
+#### MessageAttachment#contentType
+
+The media type of a MessageAttachment.
+
+### MessageButton
+
+A builder class which makes constructing button type Message Components easy.
+
+### MessageComponentInteraction
+
+Provides gateway support for receiving interactions from Message Components. Subclass of `Interaction`.
+
+### MessageManager
+
+Methods were added to `MessageManager` to provide API support for uncached messages.
+
+#### MessageManager#crosspost
+
+`channel.messages.crosspost('765432109876543210')`
+
+Equivalent to `message.crosspost()`.
+
+#### MessageManager#edit
+
+`channel.messages.edit('765432109876543210', content, options)`
+
+Equivalent to `message.edit(content, options)`.
+
+#### MessageManager#pin
+
+`channel.messages.pin('765432109876543210', options)`
+
+Approximately equivalent to `message.pin(options)` but does not resolve to a Message.
+
+#### MessageManager#react
+
+`channel.messages.react('765432109876543210', emoji)`
+
+Approximately equivalent to `message.react(emoji)` but does not resolve to a MessageReaction.
+
+#### MessageManager#unpin
+
+`channel.messages.unpin('765432109876543210', options)`
+
+Approximately equivalent to `message.unpin(options)` but does not resolve to a Message.
+
+### MessageMentions
+
+#### MessageMentions#repliedUser
+
+Checks if the author of a message being replied to has been mentioned.
+
+### MessagePayload
+
+This class has been renamed from APIMessage.
+
+Global headers can now be set in the HTTP options.
+
+### MessageSelectMenu
+
+A builder class which makes constructing select menu type Message Components easy.
+
+### NewsChannel
+
+#### NewsChannel#addFollower
+
+`channel.addFollower(channel)`
+
+Provides API support for bots to follow announcements in other channels.
+
+#### NewsChannel#setType
+
+`channel.setType('text')`
```

Comment:
Is this still 'text' or has it been changed to
```suggestion
`channel.setType('GUILD_TEXT')`
```
?

Reply:
`GUILD_TEXT` is correct